hex/spec/dependabot/hex/file_updater_spec.rb

```diff
@@ -177,6 +177,81 @@
           expect(updated_business_content).to include(%({:plug, "~> 1.4.0"},))
         end
       end
+
+      context "with sub projects" do
+        let(:mixfile_fixture_name) { "sub_projects" }
+        let(:lockfile_fixture_name) { "sub_projects" }
+
+        let(:files) { [mixfile, lockfile, sub_mixfile1, sub_mixfile2] }
+        let(:sub_mixfile1) do
+          Dependabot::DependencyFile.new(
+            name: "apps/dependabot_business/mix.exs",
+            content: fixture("mixfiles", "dependabot_business")
+          )
+        end
+        let(:sub_mixfile2) do
+          Dependabot::DependencyFile.new(
+            name: "apps/dependabot_web/mix.exs",
+            content: fixture("mixfiles", "dependabot_web")
+          )
+        end
```

Comment:
Same, once the project is set up: 
```suggestion
        let(:files) { project_dependency_files("umbrella_sub_projects") }
```